gulp/tasks/scripts.js

```diff
@@ -1,14 +1,45 @@
 'use strict';
-var stream        = require('../utils/browserSync').stream;
+var stream  = require('../utils/browserSync').stream;
+var webpack = require('webpack');
+var gulpWebpack = require('webpack-stream');
+var glob = require('glob');
 
 module.exports = function (gulp, $, config) {
-  var srcFiles   = config.appFiles.scripts;
+  var tasksHelper  = require('../utils/tasksHelpers')(gulp, config);
+  var scriptsFiles = config.appFiles.scripts;
+  var srcFiles = glob.sync(scriptsFiles);
   var destPath   = config.paths.scripts.dest;
+  var skeletonRoot = config.basePaths.root;
+  var srcRoot = config.basePaths.src;
 
   var task = function () {
-    return gulp.src(srcFiles)
+    return gulp.src(scriptsFiles)
       .pipe($.eslint({envs: ['browser']}))
       .pipe($.eslint.format())
+      .pipe(gulpWebpack({
+        debug: true, //TODO improve this one we have env depending builds
```

Comment:
Cause I don't want half temp solutions for the skeleton.
Will address this in #84